tests/Functional/Fixtures/Symfony/AppCache.php

```diff
@@ -25,4 +29,14 @@ public function handle(Request $request, $type = HttpKernelInterface::MASTER_REQ
 
         return $response;
     }
+
+    /**
+     * Made public to allow event subscribers to do refresh operations.
+     *
+     * {@inheritDoc}
+     */
+    public function fetch(Request $request, $catch = false)
```

Comment:
strange that a trait can't change visibility of a method - or is there a way that would allow it?